README.md

```diff
@@ -0,0 +1,198 @@
+<a name="readme-top"></a>
```

Comment:
- [ ] OPTIONAL:  All the furniture in the homepage desktop view is duplicated, even if I add more it will get duplicated too:

Firefox:

![image](https://github.com/anbehindY/furniture-mall-frontend/assets/57421823/d1384900-4d4d-4af7-83b0-3a164da3920e)

Chrome:

![image](https://github.com/anbehindY/furniture-mall-frontend/assets/57421823/153c86bd-ab4e-4ebd-a74e-b7b3ac21d694)

Kindly, make it so each furniture piece is listed just once, and not duplicated. 💪